lib/test_queue/runner.rb

```diff
@@ -337,6 +349,7 @@ def distribute_queue
               # If we have a slave from a different test run, don't respond, and it will consider the test run done.
               sock.write("OK\n")
               remote_workers += num
+              quorum_members += num
```

Comment:
I also didn't want to end up in a case where some workers are stopped and others are finishing up and then we hit the quorum timeout and erroneously abort the build.